src/mumble/Messages.cpp

```diff
@@ -319,7 +319,7 @@ void MainWindow::msgUserState(const MumbleProto::UserState &msg) {
 		if (msg.has_self_deaf())
 			pDst->setSelfDeaf(msg.self_deaf());
 
-		if (pSelf && pDst != pSelf && (pDst->cChannel == pSelf->cChannel)) {
+		if ((pSelf && pDst != pSelf && (pDst->cChannel == pSelf->cChannel)) || (pSelf && pDst != pSelf && ((pDst->cChannel->allLinks().contains(pSelf->cChannel))))) {
```

Comment:
Can we simplify this one to be like the ones below?

I.e., something like:

    if ((pSelf && pDst != pSelf && (pDst->cChannel == pSelf->cChannel) || (pDst->cChannel->allLinks().contains(pSelf->cChannel))))) {